waku/v2/utils/wakuswitch.nim

```diff
@@ -14,6 +15,40 @@ import
 proc withWsTransport*(b: SwitchBuilder): SwitchBuilder =
   b.withTransport(proc(upgr: Upgrade): Transport = WsTransport.new(upgr))
 
+proc getSecureKey(path : string): TLSPrivateKey
+  {.raises: [Defect,TLSStreamProtocolError, IOError].} =
+  var stringkey : string = readFile(path)
+  ##echo stringkey
+  try :
+    let key: TLSPrivateKey = TLSPrivateKey.init(stringkey)
```

Comment:
```suggestion
    let key = TLSPrivateKey.init(stringkey)
```
<-- Minor suggestion.
There's no need to specify the `type` when it's obvious from initialisation and we generally don't in `nim-waku.